bellows/types/named.py

```diff
@@ -209,8 +229,16 @@ class EzspConfigId(basic.uint8_t, enum.Enum):
     CONFIG_ZLL_GROUP_ADDRESSES = 0x2F
     # ZLL rssi threshold initial configuration.
     CONFIG_ZLL_RSSI_THRESHOLD = 0x30
+    # ??? - TODO.
+    CONFIG_RF4CE_PARING_TABLE_SIZE = 0x31
+    # ??? - TODO.
```

Comment:
Docs have: "The maximum number of outgoing
RF4CE packets supported by the
stack. "